lang/cs/Org.Apache.REEF.Tests/Functional/IMRU/TestFailMapperEvaluatorsWithFailedResultHandlerOnDispose.cs

```diff
@@ -67,7 +67,11 @@ public override void TestFailedMapperOnLocalRuntime()
             Assert.Equal(numTasks, runningTaskCount);
 
             // Tasks should fail or complete or disappear with failed evaluator
-            Assert.Equal(numTasks, completedTaskCount + failedEvaluatorCount + failedTaskCount);
+            // However, as TaskRuntime Inform the driver about the result before throw exception caused by the dispose in 
+            // the result handler, that may end up two events from the update task/evaluator, one is CompletedTask, 
+            // the other is FailedEvaluator
+            int total = completedTaskCount + failedEvaluatorCount + failedTaskCount;
+            Assert.True(numTasks == total || numTasks == total - 1);
```

Comment:
Please add a message containing the actual number of received messages as well as the expected number.